### DISCUSSION TITLE
Background Images?

### DISCUSSION THREAD

**Discussion by CheapThrowawayPanda** [stoatchat/discussions#450] [Feature Suggestions]
It would be cool to implement Background Image's like BetterDiscord.

Some examples:
![image](https://user-images.githubusercontent.com/116316894/197070341-1ac64e23-3ba0-444f-80d1-9829ca93518e.png)
![image](https://user-images.githubusercontent.com/116316894/197070377-b9e33e4f-5beb-45c5-b908-d29dc0891863.png)

**Comment by ghost** [comment]
You can use custom CSS within the client to add background images.

Should I make a Gist with a snippet on how to do it?
